packages/legend-application-query/src/components/Core_LegendQueryApplicationPlugin.tsx

```diff
@@ -463,24 +463,47 @@ export class Core_LegendQueryApplicationPlugin extends LegendQueryApplicationPlu
                     }
                   }
                 }
+              } else {
+                if (queryBuilderState.changeDetectionState.hasChanged) {
```

Comment:
i think here we can just check if the rawLambda is not stub right ?
then they will have changes

Reply:
export const isStubbed_RawLambda = (rawLambda: RawLambda): boolean =>
  !rawLambda.body && !rawLambda.parameters;

May I know if you are referring to this function above?

If so, I don't think it works because the rawLambda body is not undefined as it has getAll()